### PR TITLE
[#165377489] Put icon for empty message list

### PR DESCRIPTION
**Pull request by lmorelli986**
This PR aims to add an icon into message list screen when it's empty.

screen3

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🐞 [#165377489](https://www.pivotaltracker.com/story/show/165377489): Come CIT voglio vedere una icona e un messaggio nel caso la inbox dei messaggi sia vuota

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/components/messages/MessagesInbox.tsx [hunk 1]
```diff
@@ -74,6 +78,14 @@ const generateMessagesStateNotArchivedArray = (
     []
   );
 
+const ListEmptyComponent = (
+  <View style={styles.emptyListWrapper}>
+    <Image
+      source={require("../../../img/messages/empty-message-list-icon.png")}
```
Comment (cloudify) [inline ts/components/messages/MessagesInbox.tsx]: why we have both the png and the svg? can't we just use the svg?

Comment (lmorelli986) [reply ts/components/messages/MessagesInbox.tsx]: Currently we cannot use svg without using further libraries (like for example [react-native-svg-uri](https://github.com/vault-development/react-native-svg-uri)), would you prefer me to remove the svg file from this branch at the moment?

**Comment by matteodesanti** [comment]
@lmorelli986 can you please also ad these messages in the screen?
empty_Inbox_messages
I don't see them in the gif.
Referring to SVG or PNG, if integrating the library to use SVG is a problem, we can go for PNG: please remove unused assets.

**Comment by lmorelli986** [comment]
svg image still missing

**Comment by cloudify** [comment]
@lmorelli986 need to compress the images with `pngcrush`

**Pull request merged**